tests/unit/test_merger.py

```diff
@@ -272,3 +272,55 @@ def test_real_record_merge_regression_1_authors_mismatch_on_update():
     assert sorted(conflict, key=itemgetter("path")) == sorted(
         expected_conflicts, key=itemgetter("path")
     )
+
+
+def test_titles_change():
+    root = {}
+
+    head = {"titles": [{"source": 1, "title": "Title 1"}, {"source": 2, "title": "Title 2"}]}
+
+    update = {"titles": [{"source": 3, "title": "Title 3"}]}
+
+    merged, conflict = merge(root, head, update)
+    expected_conflicts = [{'path': '/titles/0', 'op': 'remove', 'value': {'title': 'Title 3', 'source': 3}, '$type': 'REMOVE_FIELD'}]
+    assert conflict == expected_conflicts
```

Comment:
why don't you `assert` anything on `merged`?

Reply:
It didn't seem important but you are right. I should also check `merged`. :+1: